frameworks/moveos-stdlib/sources/event_queue.move

```diff
@@ -27,16 +29,31 @@ module moveos_std::event_queue {
         emit_time: u64,
     }
 
+    /// The off-chain event
+    /// Every on-chain event also trigger an off-chain event
+    struct OffChainEvent<E> has copy, drop, store {
```

Comment:
What is the different between OnChainEvent and OffChainEvent?

Reply:
OffChainEvent emit via `event::emit`, OnChainEvent store on chain.